content/docs/2.0/scalers/metrics-api.md

```diff
@@ -0,0 +1,80 @@
++++
+title = "Metrics API"
+layout = "scaler"
+availability = "v2.0+"
+maintainer = "Community"
+description = "Scale applications based on a metric provided by an API"
+go_file = "metrics_api_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `metrics-api` trigger that scales based on a metric value provided by an API. 
+
+This scaler allows users to utilize **any existing APIs** as a metric provider.  
```

Comment:
Bold to emphasize than any API mean in fact any API, so one can scale their deployment using current day length from https://ipgeolocation.io/astronomy-api.html 😃